charts/substra-backend/UPGRADE.md

```diff
@@ -1,5 +1,9 @@
 ## unreleased
 
+## 23.0.0
+
+If you have set `worker.rbac.create=false`, you now need to also set `worker.serviceAccount.create=false` to preserve the same behavior.
```

Comment:
Do we envision cases where we only create a SA without binding it?

Reply:
No, this behavior is just the simplest to implement.

What do you think would be best? If RBAC is off then SA creation is automatically disabled as well?